main.go

```diff
@@ -1,57 +1,43 @@
 package main
 
 import (
-	"log"
-	"os"
-	"os/signal"
-	"strconv"
-	"syscall"
+	l "log"
 
-	"github.com/deis/logger/syslogish"
+	"github.com/deis/logger/log"
+	"github.com/deis/logger/osutils"
+	"github.com/deis/logger/storage"
 	"github.com/deis/logger/weblog"
 )
 
 var (
 	// TODO: When semver permits us to do so, many of these flags should probably be phased out in
 	// favor of just using environment variables.  Fewer avenues of configuring this component means
 	// less confusion.
-	storageType = getopt("STORAGE_ADAPTER", "memory")
-	numLines, _ = strconv.Atoi(getopt("NUMBER_OF_LINES", "1000"))
+	storageType    = osutils.GetStringOpt("STORAGE_ADAPTER", "memory")
+	numLines, _    = osutils.GetIntOpt("NUMBER_OF_LINES", 1000)
+	aggregatorType = osutils.GetStringOpt("AGGREGATOR_TYPE", "nsq")
 )
 
 func main() {
-	syslogishServer, err := syslogish.NewServer(storageType, numLines)
+	storageAdapter, err := storage.NewAdapter(storageType, numLines)
 	if err != nil {
-		log.Fatal("Error creating syslogish server", err)
+		l.Fatal("Error creating storage adapter:", err)
 	}
-	weblogServer, err := weblog.NewServer(syslogishServer)
+
+	aggregator, err := log.NewAggregator(aggregatorType, storageAdapter)
 	if err != nil {
-		log.Fatal("Error creating weblog server", err)
+		l.Fatal("Error creating log aggregator", err)
```

Comment:
Yep. I always thought `log.Fatal` inserted at least a space between the message and the error, but I was obviously wrong about that. I will fix.